wallet-sdk/src/test/kotlin/org/stellar/walletsdk/CustomerManagementTest.kt

```diff
@@ -0,0 +1,67 @@
+package org.stellar.walletsdk
+
+import io.ktor.client.plugins.*
+import io.ktor.http.*
+import kotlin.test.assertNotNull
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.stellar.walletsdk.horizon.SigningKeyPair
+
+class CustomerManagementTest {
+  private val wallet =
+    Wallet(
+      StellarConfiguration.Testnet,
+      ApplicationConfiguration { defaultRequest { url { protocol = URLProtocol.HTTP } } }
```

Comment:
You don't need to specify http scheme here

Reply:
I got it from another test